css/components/footer.css

```
@@ -0,0 +1,29 @@
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 160px;
+  background-color: #111827;
+}
+
+.footer-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 1120px;
+  color: #ffffff;
+  font-size: 16px;
+  line-height: 19.09px;
```

💊 제안
해당 line-height 값을 아마 figma에서 보고 추가하셨을 것 같아요.
다만 아래 속성값은 사실 의미가 없을 가능성이 큽니다.
피그마는 디자인을 보고 css 값들을 나열하고, 해당 값이 실제로 구현하는데 필수적인지는 알 수 없습니다.
그래서 피그마 속성값을 그대로 쓰시기보다 참고만해주세요~
그리고 line-height 값이 필요하시더라도 정수로 사용하시는 것이 좋습니다~

```suggestion
  line-height: 19px; 
```